etc/skeleton/native/native.sh

```diff
@@ -1,3 +1,9 @@
-#!/usr/bin/env sh
-echo "TODO: native driver"
+#!/usr/bin/env bash
+while read -r req
```

Comment:
Can you please add a comment outlining briefly what this file does? E.g.,

```
# This implements a trivial native driver that returns its input unchanged
```

or words to that effect.

Reply:
Good catch, thanks! I definitely wanted to add a TODO for this.